src/initialization/MOM_shared_initialization.F90

```diff
@@ -1135,7 +1155,8 @@ subroutine write_ocean_geometry_file(G, param_file, directory, geom_file)
   vars(22) = var_desc("Dblock_v","m","Blocked depth at v points",'v','1','1')
   vars(23) = var_desc("Dopen_v","m","Open depth at v points",'v','1','1')
 
-  nFlds_used = 19 ; if (G%bathymetry_at_vel) nFlds_used = 23
+
+  nFlds_used = 21 ; if (G%bathymetry_at_vel) nFlds_used = 25
```

Comment:
@wfcooke Looks better. Could you remove this last residual change? I think the original values of 19 and 23 are correct. Thx.